RussianFilter/sections/specific.txt

```diff
@@ -23,6 +23,7 @@ kino-2017.online###Adforce-600x300
 pravoved.ru##.feed-bnnrs--right
 pravoved.ru##div[data-banners^="R-A-"]
 reporter-ua.com##.e-banner
+||vgtimes.ru/engine/classes/min/index.php
```

Comment:
It seems that this rule breaks website, for example "search" button doesn't work.

By the way, if I'm not wrong, they use some kind of anti-adblock script and display different ads for adblock users, these rules fix it on my end:
`vgtimes.ru#$#.adsbox#adfox_ { display: block!important; }`
`vgtimes.ru#%#AG_setConstant('adBlockEnabled', 'false');`